GregWeng.md

```diff
@@ -1,3 +1,18 @@
+## 08/10 ~ 08/14
+
+LockScreen: fix interrmitent errors on Mulet
+
+* Still tracing the possible root case, and still can't successfully reproduce it on my local machine, no matter on Mac or Ubuntu
+  * As a result I can only debug it on treeheader per half hour (modifying, pushing and collecting results)
+  * Will keep trying to reproduce it on local consoles, but I have no further ideas; may ask Alexandre his best strategy
+
+Performance Tools:
+* Bug 1186762 - [Build][Raptor Transformer] Simplify the command to make Raptor + transforming process (r+)
+  * Set checkin-needed but not get landed yet. Will land it manually later.
+
+mozSettings Performance Tuning:
```

Comment:
We don't even know what to do now, especially if I need to respect the date of this record (8/14). But today (8/18) we finally got some concrete **failures** that proven our guessing looks like wrong. I will update it on the next week report. But if you don't mind the date and a failure of guessing is valuable, I can update it here.